traffic_ops/traffic_ops_golang/ping/keys.go

```diff
@@ -20,25 +20,29 @@ package ping
  */
 
 import (
-	"errors"
+	"github.com/apache/trafficcontrol/lib/go-tc"
 	"net/http"
 
 	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/api"
 	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/riaksvc"
 )
 
 func Keys(w http.ResponseWriter, r *http.Request) {
+	alerts := tc.CreateAlerts(tc.WarnLevel, "This endpoint is deprecated")
```

Comment:
since this is being deprecated because the `/riak/ping` endpoint did the same what do you think about putting in an alternative for this like we did when we moved that one? `"text": "This endpoint is deprecated, please use GET /api/2.0/vault/ping instead"`

Reply:
you should be able to use the CreateDeprecationAlerts function in api.go to keep that deprecation message consistent